main.tf

```diff
@@ -45,8 +53,8 @@ resource "aws_ecs_task_definition" "default" {
   network_mode             = var.network_mode
   cpu                      = var.task_cpu
   memory                   = var.task_memory
-  execution_role_arn       = length(local.task_exec_role_arn) > 0 ? local.task_exec_role_arn : join("", aws_iam_role.ecs_exec.*.arn)
-  task_role_arn            = length(local.task_role_arn) > 0 ? local.task_role_arn : join("", aws_iam_role.ecs_task.*.arn)
+  execution_role_arn       = length(local.task_exec_role_arn) > 0 ? local.task_exec_role_arn : join("", aws_iam_role.ecs_exec[*].arn)
+  task_role_arn            = length(local.task_role_arn) > 0 ? local.task_role_arn : join("", aws_iam_role.ecs_task[*].arn)
```

Comment:
```suggestion
  execution_role_arn       = length(local.task_exec_role_arn) > 0 ? local.task_exec_role_arn : aws_iam_role.ecs_exec[0].arn
  task_role_arn            = length(local.task_role_arn) > 0 ? local.task_role_arn : aws_iam_role.ecs_task[0].arn
```

Reply:
as long as we won't run into issues I would prefer to keep it like in the source

Reply:
i think i've seen errors when trying to do it like so and since the count is always 1 when it exists, there is no need for `join` + `*`

Reply:
As we've discussed in the meeting, i'm fine with keeping it as-is to stay compatible and have it easy to pull in upstream changes from cloudposse